bundles/turn-combat/commands/attack.js

```diff
@@ -4,10 +4,22 @@ const { Broadcast: B } = require('ranvier');
 const CombatController = require('../lib/CombatController');
 
 module.exports = {
-  command: state => (args, player) => {
-    const target = [...player.room.npcs][0];
-    const controller = new CombatController(state);
-
-    controller.beginCombat(player, target);
-  }
+    command: state => (args, player) => {
+        const target = [...player.room.npcs][0];
+        const controller = new CombatController(state);
+        //TODO: Add pvp flag to this
```

Comment:
For the sake of keeping PRs small I would consider adding this fix as a separate PR particularly as it has several TODOs. Smaller PRs mean a higher chance for each individual one to get merged in sooner than later.

Reply:
Agreed, I just found another bug with it, so I'll back this one out of the PR